### DISCUSSION TITLE
Elenai Dodge steel feathers and sprinting config

### DISCUSSION THREAD

**Discussion by Uub112** [ACGaming/UniversalTweaks#364] [Tweak Suggestions]
Elenai Dodge 2's feather resource has this [steel feather](https://github.com/ElenaiDev/ElenaiDodge2.0/wiki/Steel-Feathers) mechanic where if you wear armor, it can turn some of your feathers bar into steel feathers, which can't be consumed to perform dodges and so giving the armor wearer an effectively smaller feathers bar.
Currently, these steel feathers can still be consumed by sprinting if the config is set to require feathers for sprinting, but it'd be neat to have an config option to make it that sprinting couldn't consume these steel feathers, making it more absolute how much less mobility you have with heavier armor setups.

**Comment by NoMoreUsernames999** [comment]
Second this. Paired with armours which reduce speed based on protection value, it really adds to the penalty of having to decide over defense or agility.
